detection-rules/attachment_file_with_unc_path.yml

```diff
@@ -0,0 +1,18 @@
+name: "Attachment: Exploitable file type with UNC path"
+description: |
+  Attached file contains a UNC path and is known to potentially relay NTLM password hashes. File extensions listed in this rule are confirmed to work on a patched Windows 11 system, as of April 2023.
+references:
+  - "https://github.com/Greenwolf/ntlm_theft"
+type: "rule"
+severity: "low"
+source: |
+  type.inbound 
+  and any(attachments, .file_extension in ("asx", "doc", "docx", "jnlp", "m3u", "pdf", "wax", "xls", "xlsx", "xml")
+      and any(file.explode(.), 
+          any(.scan.strings.strings, regex.icontains(., '\\\\([a-z0-9_%.$-]+)\\([a-z0-9_%.$-]+)') and (regex.icontains(., '(?:[0-9]{1,3}\.){3}[0-9]{1,3}') or regex.icontains(., '([a-z0-9]+(-[a-z0-9]+)*\.)+[a-z]{2,}'))) // normal UNC paths with IP or domain
+          or any(.scan.strings.strings, regex.icontains(., '%5C%5C([a-z0-9_%.$-]+)%5C([a-z0-9_%.$-]+)')) // percent-encoded UNC paths
+      )
+  )
```

Comment:
```suggestion
  type.inbound 
  and any(attachments, .file_extension in ("asx", "doc", "docx", "jnlp", "m3u", "pdf", "wax", "xls", "xlsx", "xml")
      and any(file.explode(.), 
          any(.scan.strings.strings, regex.icontains(., '\\\\([a-z0-9_%.$-]+)\\([a-z0-9_%.$-]+)') 
              and (regex.icontains(., '(?:[0-9]{1,3}\.){3}[0-9]{1,3}')
              // normal UNC paths with IP or domain
              or regex.icontains(., '([a-z0-9]+(-[a-z0-9]+)*\.)+[a-z]{2,}')))
              // percent-encoded UNC paths
          or any(.scan.strings.strings, regex.icontains(., '%5C%5C([a-z0-9_%.$-]+)%5C([a-z0-9_%.$-]+)')) 
      )
  )
```

Reply:
Tried to update the formatting for readability, moved the comments before the lines they were decorating. I don't think I broke anything but please validate :)